SPECS/unbound/unbound.spec

```diff
@@ -10,10 +10,6 @@ URL:            https://nlnetlabs.nl/projects/unbound/about/
 #Source0:       https://github.com/NLnetLabs/%{name}/archive/release-%{version}.tar.gz
 Source0:        %{name}-release-%{version}.tar.gz
 Source1:        %{name}.service
-# CVE-2020-12662.patch also fixes CVE-2020-12663
```

Comment:
Same as other spec. Please remove corresponding patch files in SPEC folder

Reply:
Done